roles.go

```diff
@@ -94,6 +94,21 @@ func (c *Client) GetRealmRoleByID(accessToken string, realmName string, roleId s
 	return resp, err
 }
 
+// UpdateRealmRole updates a specific realm role’s representation
+// PUT /auth/admin/realms/demorealm/roles-by-id/f19e86ad-ddf2-4397-9a36-63bf02119fe8
+// {
+//     "id": "f19e86ad-ddf2-4397-9a36-63bf02119fe8",
+//     "name": "offline_access",
+//     "description": "${role_offline-access}",
+//     "composite": false,
+//     "clientRole": false,
+//     "containerId": "b0b76f0e-3405-4d43-97da-4556d4cff122",
+//     "attributes": {}
+// }
+func (c *Client) UpdateRealmRoleByID(accessToken string, realmName string, roleId string) error {
+	return c.put(accessToken, url.Path(roleByIDPath), url.Param("realm", realmName), url.Param("id", roleId))
```

Comment:
doesn't this need a body?

Reply:
Yes! The body is what is commented on top looking for examples